### DISCUSSION TITLE
Copilot student sign up time?

### DISCUSSION THREAD

**Discussion by Geronium234** [community/community#57351] [GitHub Education]
I'm not sure if this has happened to anyone else but I signed up for the student copilot and my friends did as well so we could use it for projects over the summer and we all put our transcripts in at around the same time but for some reason all of them got in and i've been waiting for a day and nothing when I put it in before some of them. Same location, same school, and same proof of school identity put in. Not sure what to do because I need access to it soon because i'm falling behind.

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
